grpc-api/src/main/java/io/deephaven/grpc_api/table/ops/EmptyTableGrpcImpl.java

```diff
@@ -31,84 +31,17 @@ public EmptyTableGrpcImpl() {
         super(BatchTableRequest.Operation::getEmptyTable, EmptyTableRequest::getResultId);
     }
 
-    @SuppressWarnings("rawtypes")
-    private List<Class> getClassTypes(final EmptyTableRequest request) {
-        final List<Class> types = new ArrayList<>();
-        for (final String typeName : request.getColumnTypesList()) {
-            try {
-                types.add(ColumnTypes.mapColumnType(typeName));
-            } catch (final IllegalArgumentException iae) {
-                throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "Error mapping column type " + typeName + ".");
-            }
-        }
-        return types;
-    }
-
     @Override
     public void validateRequest(final EmptyTableRequest request) throws StatusRuntimeException {
-        //noinspection rawtypes
-        final List<Class> types = getClassTypes(request);
-        final List<String> names = request.getColumnNamesList();
-        if (types.size() != names.size()) {
-            throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "Inconsistent number of column names and types.");
+        if (request.getSize() < 0) {
+            throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "Size must be greater than zero");
         }
     }
 
     @Override
     public Table create(final EmptyTableRequest request, final List<SessionState.ExportObject<Table>> sourceTables) {
         Assert.eq(sourceTables.size(), "sourceTables.size()", 0);
```

Comment:
`eqZero` is even more idiomatic, or `Assert.assertion(sourceTables.isEmpty(), "sourceTables.isEmpty()")`